sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/apachecommons/lang/StringUtils.java

```diff
@@ -1392,4 +1392,28 @@ public static boolean isWhitespace(final CharSequence cs) {
     public static String defaultString(final String str, final String defaultStr) {
         return str == null ? defaultStr : str;
     }
+
+    public static String deleteWhitespace(String str) {
```

Comment:
where did we copy this from? We should also have the documentation for this (might be helpful in case of debugging)